backends/gdx-backend-moe/src/com/badlogic/gdx/backends/iosmoe/IOSPreferences.java

```diff
@@ -184,14 +184,17 @@ private NSString convertKey (String key) {
 	}
 
 	@Override
-	public void flush () {
+	public boolean flush () {
+		final boolean[] success = {false};
 		ObjCRuntime.autoreleasepool(new Runnable() {
 			@Override
 			public void run() {
 				if (!nsDictionary.writeToFileAtomically(file.getAbsolutePath(), false)) {
 					Gdx.app.debug("IOSPreferences", "Failed to write NSDictionary to file " + file);
+					success[0] = true;

```

Comment:
Please recheck the condition here, having `success=true` and `log("..failed..")` in the same block seems wrong.